### ISSUE TITLE
renderOptions always empty

### ISSUE THREAD

**Issue by tonioseiler** [verbb/formie#1850]
### Describe the bug

In my custom template, I want to override a specific field type. For this, I pass render options into the rendered form:

### Steps to reproduce

1. 
2. 

```
{% set renderOptions = {
                formNotificationRecipients: block.formNotificationRecipients
            } %}

            {{ craft.formie.renderForm(block.formieFormHandle, renderOptions) }}
```

In the fieled, my code looks like this:

```
{% if value is empty %}
        {% set mail = renderOptions.formNotificationRecipients ?? '' %}
    {% else %}
        {% set product = craft.entries.section('products').slug(value).one() %}
        {% if product %}
            {% set mail = product.formNotificationRecipients %}
        {% endif %}
    {% endif %}

    {{ fieldtag('fieldInput', {
        value: mail ?? false,
    }) }}
```

The field is always emty. When I prompt the renderOptions, its always an empty array.

Thanks

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

### Craft CMS version

Craft Pro 4.7.3

### Plugin version

2.1.10

### Multi-site?

Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.12"`.

**Comment by tonioseiler** [comment]
Works perfect. Thank you!

–––––––––––––––––––
Tonio Seiler
Dipl. Ing. FH Informatik
***@***.***

Furbo GmbH
Rüdigerstrasse 10
8045 Zürich

+41 44 501 85 65
www.furbo.ch

Abwesenheit: Mittwochs

**Comment by engram-design** [comment]
Fixed in [2.1.13](https://github.com/verbb/formie/releases/tag/2.1.13)
